### ISSUE TITLE
[OFFICIAL] ParadeDB

### ISSUE THREAD

**Issue by philippemnoel** [artifacthub/hub#3550]
Hi everyone,

We've listed the ParadeDB Helm Chart in ArtifactHub, and would like to apply for official status. I've followed all the instructions, which are below. Thank you for considering us.

- **Repository name** *(in `artifacthub.io`)*: ParadeDB Helm Chart
- **Official packages** *(leave empty if all packages in the repository are official)*: 
- **Project URL: *https://github.com/paradedb/helm-charts*
- **Is the publisher a CNCF project? (graduated, incubating or sandbox): *No*
- **Source code URL: *https://github.com/paradedb/paradedb*
- **Other relevant URLs: *https://paradedb.com*

Requirements:
- [x] The repository has already obtained the [Verified Publisher](https://artifacthub.io/docs/topics/repositories/#verified-publisher) status.
- [x] The user requesting the status is the publisher of the repository in Artifact Hub, or belongs to the organization publishing it.
- [x] All official packages available in the repository provide a `README.md` file with some documentation that can be displayed on Artifact Hub.

**Comment by tegioz** [comment]
Done! 🙂

**Comment by philippemnoel** [comment]
> Done! 🙂

Thank you!

**Comment by philippemnoel** [comment]
Would it please be possible to get the whole repository marked as official? Or is this not relevant when the package within it is marked as official? Apologies for the silly question

![Capture d’écran, le 2023-12-19 à 14 05 58](https://github.com/artifacthub/hub/assets/21990816/902fffd5-8026-49bd-aa61-1f7f26d0c7dd)

**Comment by tegioz** [comment]
It's not relevant actually, the result is the same in the end. At the beginning it was only possible to do it at the repository level. Then we added the ability to do it at the package level for more fine grained control. These days we almost always do it per package as we've encountered some situations in which non-official packages were added to an official repository accidentally. So we try to avoid it as much as possible 🙂

It was a good question, no problem. We'd like to remove that icon from the control panel as it's misleading.
